Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -62,6 +62,78 @@ class AppStoreConnectService {
         return BetaTester(output)
     }
 
+    func addTestersToGroup(
+        bundleId: String,
+        groupName: String,
+        emails: [String]
+    ) throws {
+        let testerIds = try emails.map {
+            try GetBetaTesterOperation(options: .init(id: nil, email: $0))
+                .execute(with: requestor)
+                .await()
+                .betaTester
+                .id
+        }
+
+        let app = try GetAppsOperation(
```

Comment:
We have a Read App Operation for a single app now don't we?

Reply:
Yes! I totally forgot that 😆